Cython/Compiler/Builtin.py

```diff
@@ -233,7 +233,7 @@ def declare_in_type(self, self_type):
     #('sum',       "",     "",      ""),
     #('sorted',    "",     "",      ""),
     #('type',       "O",    "O",     "PyObject_Type"),
-    BuiltinFunction('unichr',     "l",    "O",      "PyUnicode_FromOrdinal", builtin_return_type='unicode'),
+    BuiltinFunction('unichr',     "i",    "O",      "PyUnicode_FromOrdinal", builtin_return_type='unicode'),
```

Comment:
@scoder Sorry, but I'm slightly puzzled about this... 

The C function prototype is 
```c
PyAPI_FUNC(PyObject*) PyUnicode_FromOrdinal(int ordinal);
```
however `BuiltinFunction` was getting arg type code `"l"` (`long`)  and not `"i"` (`int`). 
Is my fix correct or am I missing something?

Reply:
My fix is correct.